ExternalLookup/src/main/java/cheetah/example/externallookup/function/ExternalLookupMapper.java

```diff
@@ -57,19 +62,19 @@ public void open(Configuration parameters) {
         client = HttpClient.newHttpClient();
 
         tokenProvider = KeyedTokenProvider.getInstance();
-        tokenProvider.registerTokenProviderIfAbsent(TOKEN_ID,
-                () -> new CachedTokenProvider(
-                        new OAuthTokenProvider(
-                                System.getenv("SERVICE_TOKEN_ENDPOINT"),
-                                System.getenv("SERVICE_CLIENT_ID"),
-                                System.getenv("SERVICE_CLIENT_SECRET"),
-                                System.getenv("SERVICE_SCOPE"))));
+        tokenProvider.registerTokenProviderIfAbsent(TOKEN_ID, () -> new CachedTokenProvider(new OAuthTokenProvider(tokenUrl, clientId, clientSecret, scope)));
 
         ParameterTool parameterTool = (ParameterTool)
                 getRuntimeContext().getExecutionConfig().getGlobalJobParameters();
-        idServiceHost = parameterTool.get("id-service-url");
+        String idServiceHost = parameterTool.get("id-service-url");
```

Comment:
is this intended to be a parameter for the taskmanager or should it be in the constructor?